src/db.c

```diff
@@ -1587,350 +1587,349 @@ static void _db_node_mask_fixup(struct db_arg_chain_tree *node)
 	node->datum &= node->mask;
 }
 
-/**
- * Generate a new filter rule for a 64 bit system
- * @param arch the architecture definition
- * @param rule the new filter rule
- *
- * This function generates a new syscall filter for a 64 bit system. Returns
- * zero on success, negative values on failure.
- *
- */
-static struct db_sys_list *_db_rule_gen_64(const struct arch_def *arch,
-					   const struct db_api_rule_list *rule)
+static int _db_rule_gen_arg_64(struct db_sys_list *s_new,
```

Comment:
Needs a comment block describing the function, see the others for an example.

Reply:
Okay, so it looks like it is added in 0936a72d1ae1 ("db: merge _db_rule_gen_64 and _db_rule_gen_32"); it would be best to add the comment block in the first appearance of the function.